src/Nav/components/NavMenuIcon/index.js

```diff
@@ -1,7 +1,7 @@
 import React, {useEffect, useRef} from 'react';
 import styles from './styles.module.css';
 
-const NavMenuIcon = ({ cb, closeIcon, isOpen, openIcon }) => {
+const NavMenuIcon = ({ cb, CloseIcon, isOpen, OpenIcon }) => {
```

Comment:
I didn't catch this the first time it was in. Can we change this to `onClick`? `cb` is not descriptive and clear.

Reply:
Fixed. Will include this in a future commit